src/server/scripts/Northrend/zone_storm_peaks.cpp

```diff
@@ -1436,6 +1436,27 @@ class spell_player_mount_wyrm : public SpellScriptLoader
         }
 };
 
+enum CollapsingCave
+{
+    SPELL_COLLAPSING_CAVE = 55486
```

Comment:
is this the same as m_spellInfo->Effects[effIndex].CalcValue() ?

Reply:
Yes. Since this is constant value I thought it could be better to keep it in the separate variable in this case.